publications-2019.bib

```diff
@@ -319,6 +319,32 @@ @Article{ZC19
   url =       {https://arxiv.org/abs/1905.00671}
 }
 
+@article{Soldner2019,
+  doi = {10.1016/j.camwa.2018.04.036},
+  url = {https://doi.org/10.1016/j.camwa.2018.04.036},
+  year = {2019},
+  month = oct,
+  publisher = {Elsevier {BV}},
+  volume = {78},
+  number = {7},
+  pages = {2183--2196},
+  author = {Dominic Soldner and Julia Mergheim},
+  title = {Thermal modelling of selective beam melting processes using heterogeneous time step sizes},
+  journal = {Computers {\&} Mathematics with Applications}
+}
+
 
+@article{Ghesmati2018,
+  doi = {10.1515/jnma-2018-0047},
+  url = {https://doi.org/10.1515/jnma-2018-0047},
+  year = {2019},
+  month = nov,
+  publisher = {Walter de Gruyter {GmbH}},
+  volume = {accepted},
+  number = {0},
+  author = {A. Ghesmati and W. Bangerth and B. Turcksin},
```

Comment:
wait, you are hiding one of your own papers in here? :-)

Reply:
What did you think? ;-)

Reply:
That you would immediately add your own papers once they are submitted/accepted. ;-)

Reply:
It's actually a bit of an aggravating case: It was accepted in 2018 and has been stuck in "published online" for nearly a year now.

But you had it on your list :-)